rootfs/api/views.py

```diff
@@ -296,6 +296,11 @@ def restart(self, *args, **kwargs):
         return Response(pagination, status=status.HTTP_200_OK)
 
 
+class AppSettingsViewSet(AppResourceViewSet):
```

Comment:
we need to implement a `get_object` method or the only way to retrieve an AppSetting will be through the UUID. See the CertificateViewSet's `get_object` method for an example.